Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,15 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import "./Header.css";
+import ImageThumbnail from "./ImageThumbnail";
+import HeaderTitle from "./HeaderTitle";
+
+const HeaderContainer = () => {
+  return (
+    <div className="container-header">
+      <ImageThumbnail />
+      <HeaderTitle />
```

Comment:
HeaderContainer should hold HeaderTitle and HeaderContent. Instead, you put HeaderContent inside of HeaderTitle, which is a little weird. It still works, but the code structure doesn't match what's implied by the names and what's displayed. I'll reference this in the other location as well.